lib/manageiq/appliance_console/external_auth_options.rb

```diff
@@ -79,11 +81,44 @@ def update_configuration(update_hash = nil)
       if update_hash.present?
         say("\nUpdating external authentication options on appliance ...")
         params = update_hash.collect { |key, value| "#{key}=#{value}" }
+        params = enable_provider_type(params)
         result = ManageIQ::ApplianceConsole::Utilities.rake_run("evm:settings:set", params)
         raise parse_errors(result).join(', ') if result.failure?
       end
     end
 
+    def validate_provider_type
+      return true unless @updates["/authentication/oidc_enabled"] == true && @updates["/authentication/saml_enabled"] == true
+      say("\Error: Both SAML and OIDC can not be enabled ...")
+      false
+    end
+
+    def enable_provider_type(params)
```

Comment:
This should be named with a `!` because you're altering the parameter passed into the method. (same with all the `provider_type_*` methods